city_block_simulation.py

```diff
@@ -129,17 +132,19 @@ def process_move(self, chosen_move_index):
                 intersection['current_timing'] += intersection['assigned_timing']
 
         self.valid_moves = self.get_valid_moves(self.current_position)
+        print("Valid move chosen.")
```

Comment:
Not a fan of prints. If we're running this 10k times, those print statements are gonna get hella annoying. Will be going through and purging the ones that are there soon

Reply:
I'll probably go through and add everything to logging which will resolve this

Reply:
Yeah, that was just a leftover from debugging something.